### PR TITLE
Follow up comments of PR #91531

### PR DESCRIPTION
**Pull request by zhaojuanmao**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #92557
* __->__ #92359

### CODE REVIEW

File: torch/distributed/fsdp/fully_sharded_data_parallel.py [hunk 1]
```diff
@@ -317,9 +317,9 @@ class FullyShardedDataParallel(nn.Module, _FSDPState):
             parameters will not be managed by this FSDP instance,
             that means these parameters will not be flattened and sharded by FSDP,
             their gradients will not be synchronized as well. With this newly added
-            argument, `ignored_modules` could be deprecated soon. For back compatiablity,
-            both `ignored_parameters` and `ignored_modules` are kept for now,
-            but only one of them could not be None.
+            argument, ``ignored_modules`` could be deprecated soon. For backward compatiablity,
```
Comment (awgu) [inline torch/distributed/fsdp/fully_sharded_data_parallel.py:320]: nit: One more typo
```suggestion
            argument, ``ignored_modules`` could be deprecated soon. For backward compatibility,
```

**Review by awgu — APPROVED** [review]
Thanks for the follow-ups!

**Comment by zhaojuanmao** [comment]
closing PR to stop merging..., found there is merge conflict..

**Pull request closed without merging**